BlockEditor/Models/Game/GameEngine.cs

```diff
@@ -34,13 +35,13 @@ public bool Pause
         public void PauseConfirmed()
         {
             Pause = false;
-            Thread.Sleep(GameEngine.FPS * 5); // wait for engine to pause
+            Thread.Sleep((int)(MsPerFrame * 5)); // wait for engine to pause
         }
 
         public GameEngine()
         {
             _timer = new System.Timers.Timer();
-            _timer.Interval = FPS;
```

Comment:
This was making it update at 37fps

Reply:
lel, i see now the 37ms FPS bug.

I was using 27ms between each frame, which gives an FPS of 37ms xD

Good job on fixing. 

Maybe we can use a slower FPS on zoom level 5%, so it lags less?
I also have another idea on fixing the lag on zoom level 5%...hmm..imma rejoin on discord so its easier to talk :p